packages/babel-parser/src/parser/statement.js

```diff
@@ -1824,6 +1824,9 @@ export default class StatementParser extends ExpressionParser {
 
     if (isFromRequired || hasSpecifiers || hasDeclaration) {
       this.checkExport(node, true, false, !!node.source);
+      if (this.hasPlugin("importAssertions") && node.assertions == null) {
+        node.assertions = [];
+      }
```

Comment:
I don't remember if our flow version supports it, but if it does we can do
```suggestion
      if (this.hasPlugin("importAssertions")) {
        node.assertions ??= [];
      }
```